src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -112,6 +138,30 @@
  * }
  * </pre>
  * <p>
+ * To configure the check to not allow {@code case, default} single-line statements without braces:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;NeedBraces&quot;&gt;
+ *  &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_CASE, LITERAL_DEFAULT&quot;/&gt;
```

Comment:
Do I need to write `&lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;false&quot;/&gt;`

Reply:
as mentioned here "[NeedBraces](https://checkstyle.org/config_blocks.html#NeedBraces)" default value of allowSingleLineStatement is false and you also asking for the same but i think you should write the property name if you have mentioned property name in below code example which you did.

Reply:
The property name cannot be mentioned without giving it a value, and that what I am saying above.

Reply:
property name with value okay but i thought u wanna know whether to write property name with default value.

Reply:
ok np